lib/bibmarkdown/document.rb

```diff
@@ -20,7 +20,7 @@ def to_markdown
         # Create the references
         refs = keys.strip.split(/\s*,\s*/).map {|key| create_reference key }
         raise "Missing reference key in #{match}" if refs.empty?
-        reflinks = refs.map{|r| r[:link]}.join ''
+        reflinks = create_element :span, "\\[" + (refs.map{|r| r[:link]}.join ", ") + "\\]", class: 'reference'
```

Comment:
`references` then maybe?

Reply:
Not sure we want a [space after the comma](https://tex.stackexchange.com/a/127037/16069), probably just `&thinsp;`.

Reply:
Ah, the CSS class actually. Because now the class `reference` would select both the group and the individuals. Variable name is fine :-)